test/dotnet.Tests/ParserTests/UninstallToolParserTests.cs

```diff
@@ -56,19 +56,19 @@ public void UninstallToolParserCanParseToolPathOption()
         public void UninstallToolParserCanParseLocalOption()
         {
             var result =
-                Parser.Instance.Parse(@"dotnet tool uninstall --local console.test.app");
+                Parser.Instance.Parse(@"dotnet tool update --local console.test.app");
```

Comment:
Could you explain this change to me?  Seems odd that we're invoking update from the uninstall tests.

Reply:
oops

Reply:
I played around the parse before hand. Forget to revert it. The counter part of these tests for update is in already